### ISSUE TITLE
`$scope.commands/overlays` do not respect visibiliy

### ISSUE THREAD

**Issue by kubouch** [nushell/nushell#4294]
It will list also symbols hidden with `hide`

**Comment by hustcer** [comment]
The aliases/commands that been hidden by `hide` were not listed now, maybe we can close this? @kubouch

**Comment by kubouch** [comment]
Yeah. Maybe later we can add the `is_hidden` column but there is a lot of columns already...

**Comment by fdncred** [comment]
I'd vote for having an `is_hidden` column because I'm not sure how you'd find them otherwise. you can always 'reject` columns if you don't want them in your output.
